changelog.txt

```diff
@@ -1,5 +1,10 @@
 This file contains old changelogs and it's refreshed periodically.
 
+= 3.1.2 = 
```

Comment:
Please look at this: https://github.com/BracketSpace/Notification/pull/124#discussion_r227333964

Also, please be more specific, so instead `[Added] date merge tag for trigger execution date.` do `[Added] Global date merge tag for trigger execution date.`